src/controllers/BanController.ts

```diff
@@ -14,8 +14,8 @@ const getBannedWord = async (): Promise<string> => {
 };
 
 const ban = async (word: string): Promise<void> => {
-    if (/[\n\t\r ]/.test(word)) {
-        throw new Error("Cannot ban a word with whitespace");
+    if (/[\n\t\r ]|\|\|/.test(word)) {
+        throw new Error("Cannot ban a word with whitespace or spoiler marks (||)");
```

Comment:
Before this change this error actually could never happen. Now that this has been introduced, can you change the call site to ensure that the error is handled?

This is bad design on my part lmao